frontend/src/components/NewRoomButton/NewRoomButton.spec.tsx

```diff
@@ -0,0 +1,26 @@
+import { describe, expect, it, vi, beforeEach } from 'vitest';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import NewRoomButton from './index';
```

Comment:
for this one and other imports, it'd be better to import it directly here like this:

```
import NewRoomButton from './NewRoomButton'
```
for consistency with other unit tests